velox/expression/FunctionSignature.h

```diff
@@ -158,6 +159,10 @@ class FunctionSignature {
     return variables_;
   }
 
+  bool orderSensitive() const {
```

Comment:
This is not a property of the signature. Rather a property of the function itself. It would be nice to decouple these.